src/main/java/redis/clients/jedis/BinaryJedis.java

```diff
@@ -171,19 +171,33 @@ public String quit() {
   }
 
   /**
+   * Test if the specified keys exist. The command returns the number of keys existed
+   * Time complexity: O(N)
+   * @param keys
+   * @return Integer reply, specifically: an integer greater than 0 if one or more keys existed
+   *         0 if none of the specified keys existed
+   */
+  public Long exists(final byte[]... keys) {
+    checkIsInMultiOrPipeline();
+    client.exists(keys);
+    return client.getIntegerReply();
+  }
+
+  /**
    * Test if the specified key exists. The command returns "1" if the key exists, otherwise "0" is
    * returned. Note that even keys set with an empty string as value will return "1". Time
    * complexity: O(1)
    * @param key
-   * @return Integer reply, "1" if the key exists, otherwise "0"
+   * @return Boolean reply, "1" if the key exists, otherwise "0"
```

Comment:
@charsyam Great for finding miss in comment! 
Btw, it doesn't return "1" or "0", so I'd like to fix it, too.